Makefile

```diff
@@ -25,6 +25,7 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
+TEST_TAGS ?= basic
```

Comment:
@praveenkumar @hferentschik My idea here was to make it easy to use - having basic test as a smoke test which you can run while on coffee break etc. And to not force dev to type a lot: `make integration TEST_TAGS=basic`, I put basic test as native option there. However I agree with you that this is not so obvious setup. Let me know about this and I will update this PR to run all tests on default.